js/src/logger.ts

```diff
@@ -2668,7 +2668,7 @@ export function initLogger<IsAsyncFlush extends boolean = true>(
   const {
     projectName,
     projectId,
-    asyncFlush,
+    asyncFlush: asyncFlushArg,
```

Comment:
While we're at it, should we update all the other places where `IsAsyncFlush` is defaulted? E.g. https://github.com/braintrustdata/braintrust-sdk/blob/8fc81c5d906bbeb49e0f3c638b3d518a9e150680/js/src/logger.ts#L3232.

Reply:
Done